python-template/{{cookiecutter.placeholder_repo_name}}/.gitignore

```diff
@@ -0,0 +1,78 @@
+*.py[cod]
+__pycache__
+.pytest_cache
+
+# C extensions
+*.so
+
+# Packages
+*.egg
+*.egg-info
+dist
+build
+eggs
+parts
+bin
```

Comment:
I'm quite suspicious of `bin` as an ignore pattern. That's a very common directory for me to see under source control.

Reply:
I'm going to punt this question for later and leave it there.